packages/tasklist-client-node-js/src/__test__/restclient.spec.ts

```diff
@@ -44,130 +41,99 @@ describe('TasklistRESTClient', () => {
 
     afterAll(() => zbc.close())
 
-    it('can request all tasks', async () => {
-        const tasklist = new TasklistRESTClient()
-        const tasks = await tasklist.getAllTasks()
-        console.log('tasks', tasks)
-        expect(tasks.length).toBeGreaterThan(0)
-    })
-
-    it('can request a task with parameters', async () => {
-        const tasklist = new TasklistRESTClient()
-        const tasks = await tasklist.getTasks({
-            state: TaskState.CREATED,
+    describe('Read operations', () => {
```

Comment:
I grouped these tests into "Read operations" and "Write operations" mostly for ease of turning on/off sets of tests while debugging. Once I had the groupings, I liked them...but this isn't critical for the changes, and if you'd like them removed I can do so.